geoportailv3/static/js/search/searchdirective.js

```diff
@@ -374,11 +345,11 @@ app.SearchDirectiveController = function($scope, $compile, gettextCatalog,
       return suggestion['translatedName'];
     },
     templates: /** @type {TypeaheadTemplates} */({
-      header: function() {
+      header: goog.bind(function() {
         return '<div class="header">' +
-            gettextCatalog.getString('Background Layers') +
+            this.gettextCatalog_.getString('Background Layers') +
             '</div>';
-      },
+      }, this),
```

Comment:
okay but then we should migrate all of our codebase to this version. ... another PR !